docs/src/app/pages/Accordion/ExampleNormal.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Accordion } from 'react-lds';
+
+const sections = [
```

Comment:
Not sure if we want to go down that road though. for a first version, this seems fine